lib/output.js

```diff
@@ -539,6 +539,7 @@ function toFormat (format, options) {
  * @param {Number} [tile.overlap=0] tile overlap in pixels, a value between 0 and 8192.
  * @param {Number} [tile.angle=0] tile angle of rotation, must be a multiple of 90.
  * @param {String} [tile.depth] how deep to make the pyramid, possible values are `onepixel`, `onetile` or `one`, default based on layout.
+ * @param {Number} [tile.skipBlank=-1] threshold to skip tile generation, a value 0 - 255 for 8-bit images or 0 - 65535 for 16-bit images
```

Comment:
I think `tile.skipBlank` should be the plural form `tile.skipBlanks`.

Please can you run `npm run docs` after fixing this, which should automagically update `docs/api-output.md`.

Reply:
Nice catch :)
Sorry I took a bit long to fix this.